src/vs/workbench/services/textfile/common/textFileEditorModel.ts

```diff
@@ -674,6 +674,10 @@ export class TextFileEditorModel extends BaseTextEditorModel implements ITextFil
 		return this.dirty;
 	}
 
+	isModified(): boolean {
+		return this.dirty;
```

Comment:
I would prefer to call `this.isDirty()` so that we make sure the 2 are always in sync.